polling_stations/apps/data_importers/management/commands/import_eoni_from_s3.py

```diff
@@ -0,0 +1,436 @@
+                    f"{x =}"
+                    f"{y =}"
+                )
+            )
+            self.stdout.write(self.style.WARNNG(f"Exception was: {str(e)}"))
+            return None, None
+
+    def reproject_row(self, row, scheme_suffix, transformer, row_num):
+        station_x_coord_field = f"{self.station_x_field_prefix}{scheme_suffix}"
+        station_y_coord_field = f"{self.station_y_field_prefix}{scheme_suffix}"
+        new_row = {field: row[field] for field in self.address_non_coord_fields}
+        address_lon, address_lat = self.reproject_coord(
+            row[self.address_x_field],
+            row[self.address_y_field],
+            transformer,
+            row_num,
+        )
+
+        station_lon, station_lat = self.reproject_coord(
+            row[station_x_coord_field],
+            row[station_y_coord_field],
+            transformer,
+            row_num,
+        )
+
+        if not all((address_lon, address_lat, station_lon, station_lat)):
+            return None
+
+        new_row[self.address_x_field.replace("COR", "4326")] = address_lon
+        new_row[self.address_y_field.replace("COR", "4326")] = address_lat
+
+        new_row.update(
+            {
+                field: row[f"{field}{scheme_suffix}"]
+                for field in self.station_non_coord_fields
+            }
+        )
+        new_row[self.station_x_field_prefix.replace("COR", "4326")] = station_lon
+        new_row[self.station_y_field_prefix.replace("COR", "4326")] = station_lat
+        return new_row
+
+    def create_reprojected_file(self):
+        scheme_suffix = self.national_suffix
+
+        if self.import_scheme == "LOCAL":
+            scheme_suffix = self.local_suffix
+
+        transformer = Transformer.from_crs("epsg:29902", "epsg:4326", always_xy=True)
+
+        new_rows = []
+        failed_rows = []
+
+        row_count = 0
+        for row_num, row in enumerate(self.reader, 1):
+            row_count += 1
+            new_row = self.reproject_row(row, scheme_suffix, transformer, row_count)
+            if new_row:
+                new_rows.append(new_row)
+            else:
+                failed_rows.append(row_count)
+
+        with open(self.output_path, mode="w", newline="") as csv_file:
+            fieldnames = new_rows[0].keys()
+            writer = csv.DictWriter(csv_file, fieldnames=fieldnames)
+            writer.writeheader()
+            writer.writerows(new_rows)
+
+        self.stdout.write(
+            self.style.SUCCESS(
+                f"Successfully created reprojected file with {len(new_rows)} rows"
+            )
+        )
+        if len(failed_rows) > 0:
+            sentry_sdk.set_context(
+                "Command",
+                {
+                    "file": f"s3://{self.bucket_name}/{self.input_key}",
+                    "error": "Reprojection error",
+                    "issues": f"Failed to reproject {len(failed_rows)} rows: {failed_rows}",
```

Comment:
Given this is a list in the other place we set it, lets make it a list here too

```suggestion
                    "issues": [f"Failed to reproject {len(failed_rows)} rows: {failed_rows}"],
```